tsdb/index/postings.go

```diff
@@ -115,6 +115,7 @@ type PostingsStats struct {
 	CardinalityLabelStats   []Stat
 	LabelValueStats         []Stat
 	LabelValuePairsStats    []Stat
+	NumLabelPairs           uint64
```

Comment:
Let's also add a comment saying it is only set for the Head block for now.

Should be simple to add for persistent block too since it just the number of entries in `Postings Offset Table`, but not a blocker for this PR, a comment would be enough for now.

```suggestion
	NumLabelPairs           int
```

Reply:
@codesome instead of comments, can I rename field `NumLabelPairs` to `NumLabelPairsInHead` or something like that?

Reply:
This is a generic struct, so let's add a comment. We will have numlabelpairs for blocks eventually.